make/Main.gmk

```diff
@@ -1310,7 +1310,11 @@ endif
 ################################################################################
 
 # all-images builds all our deliverables as images.
+ifeq ($(call isTargetOs, aix), true)
+all-images: product-images test-image all-docs-images
```

Comment:
I think you should indent the all-images line like in the other places.

Reply:
```suggestion
all-images: product-images test-image all-docs-images
ifeq ($(call isTargetOs, aix), false)
  all-images: static-jdk-image
endif
```

Reply:
Eriks suggestion is almost okay, but you should filter on the supported platforms for static images instead, that is, check `ifeq ($(call isTargetOs, linux macosx), true)` instead.

Reply:
Hi Magnus, what about Windows ?

Reply:
Static images does build but does not work on Windows. (I'm currently working on fixing this.) But sure, you can include it here as well.